--- etc/defaults/rc.conf.orig	2010-09-01 11:25:37.000000000 -0700
+++ etc/defaults/rc.conf	2010-09-29 02:07:38.660450870 -0700
@@ -438,6 +438,13 @@
 #ubthidhci_addr="2"		# Check usbconfig list to find the correct
 				# numbers for your system.
 
+### Network link/usability verification options
+netwait_enable="NO"		# Enable rc.d/netwait (or NO)
+netwait_ip=""			# IP addresses to be pinged by netwait.
+netwait_timeout="60"		# Total number of seconds to perform pings.
+netwait_if=""			# Interface name to watch link state on.
+netwait_if_timeout="30"		# Total number of seconds to monitor link state.
+
 ### Miscellaneous network options: ###
 icmp_bmcastecho="NO"	# respond to broadcast ping packets
 
